liquid.py

```diff
@@ -327,16 +389,94 @@ def get_job(hcl_path, substitutions={}):
     return template.safe_substitute(set_volumes_paths(substitutions))
 
 
+def gc():
+    """Stop collections jobs that are no longer declared in the ini file."""
+
+    nomad_jobs = nomad.jobs()
+
+    for job in nomad_jobs:
+        if job['ID'].startswith('collection-'):
+            collection_name = job['ID'][len('collection-'):]
+            if collection_name not in config.collections and job['Status'] == 'running':
```

Comment:
No need to check for `job['status'] == 'running'`, just stop them all, it won't return an error.

Reply:
Yeah, I just don't like seeing stop messages for services that were actually already stopped.

Reply:
Fair enough :)